server.py

```diff
@@ -54,6 +54,9 @@
         "134": ["0005"],
         "139": ["0001"],
         "144": ["0001", "0002"],
+        "160": ["0001"],
```

Comment:
Can you confirm this one with Tracey/Mark?  I was told that this is 0002.  The other 2 for 165 and 169 are right.

Reply:
I originally got told 0001, then it got changed without anyone telling me >_>.  I'll sort it